contracts/test/CustomDecimalERC20Mock.sol

```diff
@@ -0,0 +1,21 @@
+// solium-disable linebreak-style
+pragma solidity 0.8.6;
+
+import "openzeppelin-contracts-sol8/token/ERC20/ERC20.sol";
+
+contract CustomDecimalERC20Mock is ERC20 {
+    uint8 private immutable _decimals;
+
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        uint8 decimals_
+    ) ERC20(_name, _symbol) {
+        _mint(msg.sender, 200000000000000000000000000);
+        _decimals = decimals_;
```

Comment:
Btw, I think we should limit the max amount of decimals up to 20 or 30, because we can possibly put 100 here, and it will owerflow even uint256

Reply:
Btw, does it really fixes #170 ? 
The issue there was connected with price calculations, but the only thing changed here is the mock...

Reply:
so we figured out a way to solve the price calculations offchain, so updated the tests according the updated way of sending price.
We send price as = `parseUnits( n , fundingTokenDecimals - seedTokenDecimals + 18)`

Reply:
Hmm, Ok, what if fundingTokenDecimals is 6 and seedTokenDecimals is 24?

Reply:
Also, updated the tests to not be dependent on decimals.

Reply:
> Hmm, Ok, what if fundingTokenDecimals is 6 and seedTokenDecimals is 24?

Yes, it might be reverted. As the price we send would be `parseUnits( n , 0 )`